django/text_extractor/views.py

```diff
@@ -179,10 +184,11 @@ def submit_document(request):
                         },
                     }
                 )
-
+            formatted_total_cost = display_cad_cost(total_cost)
             context = {
                 "ocr_docs": completed_documents,
                 "user_request_id": user_request.id,
+                "total_cost": formatted_total_cost,
```

Comment:
You should be passing through the cost in the ocr_docs dict (since cost is for each ocr_doc in the table).
Currently, if you processed 5 docs (and merge is off) and the costs were $0.01, $0.02, $0.03, $0.04, $0.05, there would be 5 rows output for the 5 ocr docs, right?
But since you are only passing 1 cost, it will say that each document cost $0.15.

Reply:
good catch! fixing it.